src/main/java/com/example/book_your_seat/coupon/controller/CouponController.java

```diff
@@ -0,0 +1,47 @@
+package com.example.book_your_seat.coupon.controller;
+
+import com.example.book_your_seat.coupon.controller.dto.CouponCreateRequest;
+import com.example.book_your_seat.coupon.controller.dto.CouponIdResponse;
+import com.example.book_your_seat.coupon.controller.dto.UserCouponResponse;
+import com.example.book_your_seat.coupon.service.CouponCommandService;
+import com.example.book_your_seat.coupon.service.CouponQueryService;
+import com.example.book_your_seat.user.domain.User;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+import static com.example.book_your_seat.common.SessionConst.LOGIN_USER;
+
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/coupons")
+@RestController
+public class CouponController {
+
+    private final CouponCommandService couponCommandService;
+    private final CouponQueryService couponQueryService;
+
+    /**
+     * 쿠폰 생성 (추후 관리자 권한 추가)
+     * @param couponCreateRequest{amount, discountRate}
+     * @return couponResponse{couponId}
+     */
+    @PostMapping
+    public ResponseEntity<CouponIdResponse> createCoupon(@RequestBody @Valid CouponCreateRequest couponCreateRequest) {
+        return ResponseEntity.status(HttpStatus.CREATED).body(couponCommandService.createCoupon(couponCreateRequest));
+    }
```

Comment:
P 4~5
```suggestion
    @PostMapping
    public ResponseEntity<CouponIdResponse> createCoupon(
        @RequestBody @Valid CouponCreateRequest couponCreateRequest
    ) {
        return ResponseEntity
            .status(HttpStatus.CREATED)
            .body(couponCommandService.createCoupon(couponCreateRequest));
    }
```

저도 이렇게 줄바꿈을 해주면 좋을 것 같습니다!